apps/costcontrol/style/views/balance.css

```diff
@@ -11,15 +11,22 @@
   margin-bottom: 1rem;
   padding: 0 0 0.8rem;
   overflow: hidden;
+  display: flex;
+  align-items: center;
 }
 
 #balance-tab .overview .info {
-  transform: translateX(-4.0rem);
+  transform: translateX(calc(var(--writing-mode-offset-x-sign) * 4.0rem));
```

Comment:
I'm not in favor of using variables as they are very Gecko-specific but I know we were already using them. I would simply prefer no to use them. If it's the preferred way for the Gaia codebase, then it's OK but if not (and if the override is simple enough, consider an override instead of a var). Not a blocker.